conductr_cli/conduct_load.py

```diff
@@ -2,59 +2,64 @@
 from pyhocon.exceptions import ConfigMissingException
 from conductr_cli import bundle_utils, conduct_url, conduct_logging
 from functools import partial
+from urllib.parse import ParseResult, urlparse, urlunparse
+from urllib.request import urlretrieve
+from pathlib import Path
+
 import json
-import os
-import re
 import requests
 
-
 @conduct_logging.handle_connection_error
 @conduct_logging.handle_http_error
 @conduct_logging.handle_invalid_config
 @conduct_logging.handle_no_file
+@conduct_logging.handle_bad_zip
 def load(args):
     """`conduct load` command"""
 
-    if not os.path.isfile(args.bundle):
-        raise FileNotFoundError(args.bundle)
-
-    if args.configuration is not None and not os.path.isfile(args.configuration):
-        raise FileNotFoundError(args.configuration)
+    print('Downloading bundle...')
```

Comment:
It says `Loading bundle to ConductR...` a bit more below. This is the full output when loading a bundle:
```
➜  conductr-bundles git:(master) ✗ conduct load filesystem-server-1.0.0-501bbb6171b564a2255f22a21b5fc5302e82961909f2db43af3f576bb997c816.zip
Downloading bundle...
Loading bundle to ConductR...
Bundle loaded.
Start bundle with: conduct run 501bbb6
Unload bundle with: conduct unload 501bbb6
Print ConductR info with: conduct info
```